<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BottomButtons</class>
 <widget class="QWidget" name="BottomButtons">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>140</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1920</width>
    <height>140</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>140</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widgetButtonsLeft" native="true">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>20</y>
     <width>900</width>
     <height>100</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>900</width>
     <height>100</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>900</width>
     <height>100</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
	border:none;
}

QRadioButton::indicator{
	width:160px;
	height:100px;
	left:0px;
	right:0px;
	top:0px;
	bottom:0px;
}
QRadioButton::indicator:unchecked{	
	image: url(:/public/bottom/button_open.png);
}
QRadioButton::indicator:checked{	
	image: url(:/public/bottom/button_ing.png);
}
QRadioButton:disabled QRadioButton::indicator{
	image: url(:/public/bottom/button_close.png);
}</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>20</number>
    </property>
    <property name="leftMargin">
     <number>20</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QRadioButton" name="rbBlood">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rbPrepare">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rbTreat">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rbBloodReturn">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rbClean">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widgetButtonsRight" native="true">
   <property name="geometry">
    <rect>
     <x>1410</x>
     <y>0</y>
     <width>510</width>
     <height>140</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>140</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>140</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
	border:none;
	font-size:36px;
	color:#fff;
}

QPushButton{
	width:150px;
	height:140px;
	padding-top:10px;
	background-image: url(:/public/bottom/paramset.png);
}
QPushButton:pressed{
	background-image: url(:/public/bottom/paramset_clicked.png);
}</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="spacing">
     <number>20</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>20</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QPushButton" name="radioParamDialysis">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>透析
参数</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="radioParamUF">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>超滤
参数</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="radioParamHeparin">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>肝素
参数</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widgetTextLeft" native="true">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>45</y>
     <width>900</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>900</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>900</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	font-size:36px;
	color:#fff;
}</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <property name="spacing">
     <number>20</number>
    </property>
    <property name="leftMargin">
     <number>20</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>血路</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>准备</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>治疗</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>回血</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>清洗</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
